src/main/java/net/greatstart/services/ImageResizer.java

```diff
@@ -0,0 +1,72 @@
+package net.greatstart.services;
+
+import lombok.extern.slf4j.Slf4j;
+import net.coobird.thumbnailator.Thumbnails;
+import net.coobird.thumbnailator.geometry.Positions;
+import net.coobird.thumbnailator.resizers.Resizers;
+import org.springframework.web.multipart.MultipartFile;
+
+import javax.imageio.ImageIO;
+import java.awt.Graphics;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.ByteArrayInputStream;
+
+@Slf4j
+public class ImageResizer {
+
+    public static byte[] getImage(MultipartFile file) throws IOException {
+        log.info("Start to resize");
+        byte[] result = null;
+        BufferedImage image = convertoImage(file);
+        log.info("img width:" + image.getWidth());
+        log.info("img height:" + image.getHeight());
+        int type;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "type" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1481)